aequilibrae/paths/results/path_results.py

```diff
@@ -107,7 +127,9 @@ def reset(self) -> None:
         if self.predecessors is not None:
             self.predecessors.fill(-1)
             self.connectors.fill(-1)
-            self.skims.fill(np.inf)
+            if self.skims is not None:
+                self.skims.fill(np.inf)
+            self._skimming_array.fill(np.inf)
```

Comment:
should this be under the condition too?

Reply:
No... they are always used to instantiate some code in Cython..